sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java

```diff
@@ -113,7 +111,8 @@ public void detectLanguagesBatchStringInput() {
     public void detectSingleTextLanguage() {
         DetectedLanguage primaryLanguage = new DetectedLanguage("English", "en", 1.0);
         StepVerifier.create(client.detectLanguage("This is a test English Text"))
-            .assertNext(response -> validateDetectedLanguages(Collections.singletonList(primaryLanguage), response.getDetectedLanguages()))
+            .assertNext(response -> validateDetectedLanguages(Collections.singletonList(primaryLanguage),
```

Comment:
Consider using the `validatePrimaryLanguage` for testing the results, rather than converting them to lists and then doing the comparison.